### ISSUE TITLE
Encoding::UndefinedConversionError

### ISSUE THREAD

**Issue by raghubetina** [ericbeland/html_validation#15]
Hi! I am running into an encoding error if I have any special characters in the document, even if `tidy` itself does not complain.

`tidy` output (note the emoji  🎉 in the document):

```
Info: Document content looks like HTML5
No warnings or errors were found.

<!DOCTYPE html>
<html>
<head>
<meta name="generator" content=
"HTML Tidy for HTML5 for Apple macOS version 5.6.0">
<meta charset="utf-8">
<title>Home page</title>
</head>
<body>
<h1>This is a valid page. 🎉</h1>
</body>
</html>

About HTML Tidy: https://github.com/htacg/tidy-html5
Bug reports and comments: https://github.com/htacg/tidy-html5/issues
Official mailing list: https://lists.w3.org/Archives/Public/public-htacg/
Latest HTML specification: http://dev.w3.org/html5/spec-author-view/
Validate your HTML documents: http://validator.w3.org/nu/
Lobby your company to join the W3C: http://www.w3.org/Consortium

Do you speak a language other than English, or a different variant of
English? Consider helping us to localize HTML Tidy. For details please see
https://github.com/htacg/tidy-html5/blob/master/README/LOCALIZE.md
```

`rspec` output:

```
Randomized with seed 31177
F

Failures:

  1) Home page: Content is valid HTML
     Failure/Error: expect(page).to have_valid_html

     Encoding::UndefinedConversionError:
       "\xF0" from ASCII-8BIT to UTF-8
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/html_validation_result.rb:118:in `write'
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/html_validation_result.rb:118:in `puts'
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/html_validation_result.rb:118:in `validate'
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/html_validation_result.rb:46:in `valid?'
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/html_validation_result.rb:23:in `initialize'
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/page_validations.rb:116:in `new'
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/page_validations.rb:116:in `validation'
     # /Users/raghubetina/.rvm/gems/ruby-2.5.0/gems/html_validation-1.1.4/lib/html_validation/have_valid_html.rb:19:in `matches?'
     # ./spec/features/home_page_spec.rb:8:in `block (3 levels) in <top (required)>'

Finished in 0.0142 seconds (files took 2.78 seconds to load)
1 example, 1 failure

Failed examples:

rspec ./spec/features/home_page_spec.rb:5 # Home page: Content is valid HTML

Randomized with seed 31177
```

Thank you so much!

**Comment by ericbeland** [comment]
I pushed an update that forces encoding on the puts line that fails. it *may* fix it.
